docs/ocis/guides/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Guides"
+date: 2020-10-01T20:35:00+01:00
+weight: 8
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/ocis/guides
+geekdocFilePath: _index.md
+geekdocCollapseSection: true
+---
+
+{{< toc >}}
+
+## Step-by-step Guides
+
+These guides should extend the generic documentation and help admins to learn Infinite Scale through digestable "hands-on" sessions.
+
+### Available Guides
+
+oCIS deployments are super simple, yet there are many configurations possible for advanced setups.
+
+- {{< icon "docker" >}} [Discover oCIS with Docker]({{< ref "ocis-remote-docker" >}}) - configure domain, certificates and port
```

Comment:
hm, the file is called ocis-remote-docker, but actually describes a containerized localhost deployment ...

The only mention of domain in the actual guide is in the docker-compose docs
```suggestion
- {{< icon "docker" >}} [Discover oCIS with Docker]({{< ref "ocis-local-docker" >}}) - localhost deployment with secret initialization and persistence of config and data
```